service/routes.py

```diff
@@ -25,7 +25,9 @@
 def index():
     """ Root URL response """
     return (
-        "Reminder: return some useful information in json format about the service here",
+        # "Reminder: return some useful information in json format about the service here",
+        jsonify( name = "products", paths=url_for("index", _external=True),
```

Comment:
May change to "Product REST API Service"

Reply:
You're right. I updated the files.